static/scss/dashboard.scss

```diff
@@ -165,6 +165,20 @@ c.validation-wrapper {
         }
       }
 
+      .course-price {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+
+        .course-price-percent {
```

Comment:
This doesn't seem like the right class name to me -- there are no percents in prices.